spec/models/user_auth_token_spec.rb

```diff
@@ -283,4 +283,26 @@
     expect(lookup.auth_token_seen).to eq(true)
   end
 
+  context "suspicious login" do
+
+    let(:user) { Fabricate(:user) }
+    let(:admin) { Fabricate(:admin) }
+
+    it "is not checked when generated for non-staff" do
+      Jobs.expects(:enqueue).with(:suspicious_login, anything).never
```

Comment:
We generally want to avoid using mocks whenever possible. It will be better if we test for the actual job being en-queued. You can access the jobs by accessing `Jobs:SuspiciousLogin.jobs.size`. For testing Sidekiq jobs, https://github.com/mperham/sidekiq/wiki/Testing provides a very good overview :+1: